Lib/test/test_threading.py

```diff
@@ -123,6 +123,30 @@ def func(): pass
             thread = threading.Thread(target=func)
             self.assertEqual(thread.name, "Thread-5 (func)")
 
+    def test_args_argument(self):
+        # Using list or tuple as *args* in constructor could
```

Comment:
Would you mind add `bpo-45735` at the start? For example: bpo-45735: Using list or tuple as *args* in constructor xxx`

Reply:
Sounds good! I will do as soon.

Reply:
done✅